static/js/github-create-branch.js

```diff
@@ -31,17 +31,37 @@ const toggleSubmitDisabled = (bool) => {
 }
 
 $(document).ready(() => {
-  $("#ghServers").auiSelect2();
-  $("#ghRepo").auiSelect2();
-  $("#ghParentBranch").auiSelect2();
-
-	$("#createBranchForm").on("aui-valid-submit", (event) => {
+  $("#ghServers").auiSelect2({
+    dropdownCssClass: "aui-select2-dropdown-server"
+  });
+  $("#ghRepo").auiSelect2({
+    dropdownCssClass: "aui-select2-dropdown-repo"
+  });
+  $("#ghParentBranch").auiSelect2({
+    dropdownCssClass: "aui-select2-dropdown-parent-branch"
+  });
+  
+  $("#createBranchForm").on("aui-valid-submit", (event) => {
 		event.preventDefault();
 		createBranchPost();
 	});
-});
+}).on('keyup', '.select2-input', debounce(event => {
+  const parentContainer = $(event.target).parent().parent();
+  const userInput = event.target.value;
+
+  if (userInput) {
+    if (parentContainer.hasClass("aui-select2-dropdown-server")) {
```

Comment:
Can't we use select's component`query` event to handle this instead of relying on CSS classes?

Reply:
To use `query` we would have to use `input type="hidden"`, which would just add complexity when trying to add more options based off from searching. 

<img width="642" alt="image" src="https://user-images.githubusercontent.com/13076255/189823208-1cbe7053-f81e-4759-9489-092c645b637b.png">

Reply:
What complexities it will add if we use `input type="hidden"`? This is the recommended way to use.

Reply:
I misunderstood its usage before.
Updated to using `query`.